README.md

```diff
@@ -4,24 +4,34 @@ This is a practical workshop about the syntax and the use of the [Markdown forma
 In particular, we will focus on the [GitHub Flavored Markdown](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax), used by GitHub.
 See the full specification of the GitHub Flavored Markdown [here](https://github.github.com/gfm/).
 
-First of all, fork [this repository](https://github.com/rosedu/workshop-markdown).
-And then create a clone of your fork:
+> [!NOTE]
+> Markdown is not only used in GitHub / GitLab and other software development platforms, but also in chat message engines such as [Discord](https://discord.com/), note taking apps such as [HackMD](https://hackmd.io/) and wiki engines such as [MDwiki](https://mdwiki.info).
+> Parts of the Markdown syntax are also used on WhatsApp, MS Teams and other chat message engines, particularly for text formatting.
 
-```console
-git clone https://github.com/<your-github-username>/workshop-markdown
-# Or git clone git@github.com:<your-github-username>/workshop-markdown
-cd workshop-markdown/
-```
+## Set Up
 
-Choose one of the two clone commands above to either clone via HTTPS (personal access token is required) or via Git (public SSH key needs to be configured).
-Replace `<your-github-username>` above with your GitHub username.
+To set up the workhop environment, follow the steps:
 
-Add the `upstream` remote to point to the [upstream repository](https://github.com/rosedu/workshop-markdown):
+1. First of all, fork [this repository](https://github.com/rosedu/workshop-markdown).
+   Use the the `Fork` button on the top right of the repository page.
 
-```console
-git remote add upstream https://github.com/rosedu/workshop-markdown
-git fetch upstream
-```
+1. Create a local clone of your fork:
+
+   ```console
+   git clone https://github.com/<your-github-username>/workshop-markdown
```

Comment:
Let's just stick to SSH. HTTPS requires uaing that annoying token and is deprecated anyway.